e2e/cli-e2e/tests/collect.e2e.test.ts

```diff
@@ -9,23 +15,45 @@ describe('CLI collect', () => {
   const exampleAuditTitle = 'Disallow unused variables';
 
   /* eslint-disable @typescript-eslint/no-unused-vars */
-  const omitVariableData = ({
+  const omitVariableAuditData = ({
+    score,
+    value,
+    displayValue,
```

Comment:
I was asked by @matejchalk  to remove the data as it was variable. In the future we will implement audit details and there we will write lots of checks :)

Reply:
I understand. However, in the past only some of the results were omitted but by looking at the omitted values for the Lighthouse plugin specifically, **all results** are omitted. So the E2E test only checks if the plugin runs, not that it returns correct results.

Reply:
Off the top of my head, accessibility or cumulative layout shift metrics should not be as dynamic if we choose a FE application under our control. So their results could be used in E2E tests.

Reply:
I get your point and noted it as TODO. It is not that easy to list all no variable audits and it would add additional maintenance effort, but I'll stat to list non variable audits in a ticket.